src/constants/interfaces.ts

```diff
@@ -42,8 +42,10 @@ export type TooltipPlacement =
     | "rightBottom";
 
 export enum ButtonClass {
```

Comment:
Too lazy to check this myself - is this enum used anywhere beyond the `CustomButton` component props? If not, would it be the right idea to just export it from `CustomButton`? That's what I would do, but I don't typically make big constant files like these.